lra/coordinator/client/narayana-client/src/main/java/io/helidon/lra/coordinator/client/narayana/NarayanaClient.java

```diff
@@ -227,13 +233,14 @@ public Optional<URI> join(URI lraId,
                         throw connectionError("Unexpected coordinator response ", res.status().code());
                 }
             } catch (Exception e) {
-                throw connectionError("Unable to join LRA", e);
+                throw connectionError("Unable to join LRA " + lraId, e);
             }
         });
     }
 
     @Override
     public void leave(URI lraId, PropagatedHeaders headers, Participant p) {
+        logF("Leaving LRA " + lraId + " participant: " + p);
```

Comment:
Wrong log statement - either use parameters and template, or guard the statement